persistent-test/persistent-test.cabal

```diff
@@ -138,6 +138,9 @@ library
                    , tagged
                    , old-locale
 
+    if impl (ghc < 8.0)
+      build-depends:   semigroups == 0.18.*
```

Comment:
Fortunately we have recently dropped support for GHC < 8, so this won't be necessary :smile: